src/searchbar/SearchBar-android.js

```diff
@@ -7,13 +7,19 @@ import {
   ActivityIndicator,
   Text,
 } from 'react-native';
-import MaterialIcon from 'react-native-vector-icons/MaterialCommunityIcons';
 
 import ViewPropTypes from '../config/ViewPropTypes';
 import Input from '../input/Input';
+import renderIcon from '../helpers/renderIcon';
 
 const SCREEN_WIDTH = Dimensions.get('window').width;
 const ANDROID_GRAY = 'rgba(0, 0, 0, 0.54)';
+const defaultSearchIcon = {
```

Comment:
Maybe move this into defaultProps? Since its default for `searchIcon`

Reply:
If we move this to `defaultProps` then as long as the user pass a `searchIcon` you'll lose the object, so you won't be able to override just one key (ex: `searchIcon={{ size: 30 }}`)

Reply:
Ahh kk fair point